Automattic/jetpack
# Remove duplicate views: Show notices

Fixes https://github.com/Automattic/wp-calypso/issues/97400

## Proposed changes:

Displays a notice when a user visits the enforced WP Admin view of a removed Calypso screen for the first time.

<img width="1279" alt="Screenshot 2024-12-13 at 16 47 57" src="https://github.com/user-attachments/assets/11869c46-becd-4439-9e24-7dd24773b7cf" />

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Go to the placeholder experiment page: 22124-explat-experiment
- Assign yourself to the `treatment` variation using the bookmarklets
- Visit any of the screens that are currently enforcing the WP Admin view:
  - Posts > All Posts
  - Posts > Tags
  - Posts > Categories
  - Portfolio > All Projects
  - Testimonials > All Testimonials
- Make sure a notice shows up correctly
- Make sure the notice does not show up in all the following scenarios:
  - Already dismissed
  - When visiting the WP Admin view if your admin interface was already set to `wp-admin` (i.e. untangled)
  - When visiting the WP Admin view if your preferred view is `classic` (i.e. tangled classic)
  - When visiting the WP Admin view while being assigned to the `control` variation
